Readability within Cascade_Layers page (#24679)
mdn/content: files/en-us/learn/css/building_blocks/cascade_layers/index.md

9. inline important styles
10. transitioning styles

> Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. The first created layer has lowest precedence, and the layer created last has highest precedence amongst layers. Normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.

Reviewer: ```suggestion
Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. Normal styles in the first created layer have the lowest precedence, and normal styles in the layer created last have the highest precedence among the layers. In other words, normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.
```

Next up are any stylings declared outside of layers. The stylings in `C.css` were not imported to a layer and will override any conflicting styles from `firstLayer` and `secondLayer`.

> Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`). Animating styles have higher precedence than inline normal styles.

Reviewer: ```suggestion
Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`). 

Animating styles have higher precedence than all normal styles, including inline normal styles.
```

> Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted inversely from normal styles. Any unlayered important styles have less precedence those declared within the layers. Important styles found within layers are, again, sorted in order of creation. For important styles, the last created layer has lowest precedence, and the first created layer has highest precedence amongst layers.

Reviewer: ```suggestion
Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
```

> Inline important styles again have higher precedence than important styles declared elsewhere, and transitioning styles have the highest precedence.

Reviewer: ```suggestion
Inline important styles again have higher precedence than important styles declared elsewhere. 

Transitioning styles have the highest precedence. When a normal property value is being transitioned, it takes precedence over all other property value declarations, even inline important styles; but only while transitioning.
```

9. inline important styles
10. transitioning styles

Normal styles declared inside layers receive the lowest priority and are sorted by the order in which the layers were created. Normal styles in the first created layer have the lowest precedence, and normal styles in the layer created last have the highest precedence among the layers. In other words, normal styles declared within `firstLayer` will be overridden by any subsequent stylings on the list if any conflicts exist.

Next up are any styles declared outside of layers. The styles in `C.css` were not imported into a layer and will override any conflicting styles from `firstLayer` and `secondLayer`. Normal styles not declared in a layer always have higher precedence than normal-importance layered styles.

> Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, `C.css`).

Reviewer: ```suggestion
Inline styles are declared using the [`style` attribute](/en-US/docs/Web/HTML/Global_attributes/style). Normal styles declared in this way will take precedence over normal styles found in the unlayered and layered style sheets (`firstLayer – A.css`, `secondLayer – B.css`, and `C.css`).
```

Animating styles have higher precedence than all normal styles, including inline normal styles.

> Important styles (`!important`) take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.

Reviewer: ```suggestion
Important styles, property values that include the `!important` flag, take precedence over any styles previously mentioned on our list. They are sorted in reverse order of normal styles. Any important styles declared outside of a layer have less precedence than those declared within a layer. Important styles found within layers are also sorted in order of layer creation. For important styles, the last created layer has the lowest precedence, and the first created layer has the highest precedence among declared. layers.
```
